components/Section.js

```diff
@@ -60,11 +62,11 @@ class Section extends PureComponent {
 
       sectionheader__text: [
         styles.sectionheader__text,
-        { color: headerTextColor },
+        { color: headerTextColor, fontFamily: headerTextFont },
       ],
       sectionfooter__text: [
         styles.sectionfooter__text,
-        { color: footerTextColor },
+        { color: footerTextColor, fontFamily: footerTextFont },
```

Comment:
What about the Cell content?

Reply:
There are already some props which make customizing cell straightforward such as ```titleTextStyle``` and ```detailTextStyle```, I am adding this so the section part can be easily customized in order to keep the whole look and feel consistent.

Reply:
Ah got it!
What do you think about making it consistent with `Cell` then and adding the props `footerTextStyle` and `headerTextStyle`? 🤔

Reply:
I see the point.
I believe it's a good idea to use ```headerTextStyle``` so we can get rid of ```headerTextColor``` and ```headerTextFont```.